NotificationSystem/SendModeratorEmail.cs

```diff
@@ -57,13 +59,16 @@ public static async Task Run(
             // TODO: make better email
             string body = EmailTemplate.GetModeratorEmailBody(documentTimeStamp, location);
 
+            var timeConstraint = TimeLimiter.GetFromMaxCountByInterval(14, TimeSpan.FromSeconds(1));
```

Comment:
Where does the number 14 come from?  At least define a named constant for it instead of just hard coding the number here.

Reply:
Moved to const